pages/gallery.html

```diff
@@ -2,11 +2,37 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Nat'l Parks Gallery</title>
+  <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon">
```

Comment:
This is failing to show up since the assets folder is one level up from the current file (needs `../`)